dso-l2/src/main/java/com/tc/objectserver/handler/RetirementManager.java

```diff
@@ -163,8 +157,9 @@ private void traverseDependencyGraph(List<Retiree> toRetire, LogicalSequence com
     }
   }
 
-  public synchronized void deferRetirement(EntityMessage invokeMessageToDefer, EntityMessage laterMessage) {
+  public void deferRetirement(EntityMessage invokeMessageToDefer, EntityMessage laterMessage) {
```

Comment:
could you explain why synchronized is not needed here?

Reply:
that is a mistake will fix.